docs/monitors/processlist.md

```diff
@@ -0,0 +1,30 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# processlist
+
+ This monitor reports processlist information for Windows
+Hosts.
+
+Sample YAML configuration:
+
+```yaml
+monitors:
+ - type: processlist
+```
+
+
+Monitor Type: `processlist`
+
+[Monitor Source Code](https://github.com/signalfx/signalfx-agent/tree/master/internal/monitors/processlist)
+
+**Accepts Endpoints**: No
+
+**Multiple Instances Allowed**: Yes
```

Comment:
This should probably not have multiple instances (it is controlled by a struct tag on the embedded monitor config in the monitor config struct definition).